src/cli/Cli.ts

```diff
@@ -35,16 +37,21 @@ export class Cli {
   public currentCommandName: string | undefined;
   private optionsFromArgs: { options: minimist.ParsedArgs } | undefined;
   private commandsFolder: string = '';
+  private showHelpOnFailure: boolean = false;
```

Comment:
Rather than defining each option as a class property here, could we get it directly from the config store to avoid cluttering the class?